src/test/java/tests/elements/factory/CustomElementFactory.java

```diff
@@ -18,6 +18,7 @@
     protected Map<Class<? extends IElement>, Class<? extends IElement>> getElementTypesMap() {
         Map<Class<? extends IElement>, Class<? extends IElement>> typesMap = new HashMap<>();
         typesMap.put(ICustomElement.class, CustomElement.class);
+        typesMap.put(IWebCustomElement.class, CustomWebElement.class);
```

Comment:
do you need it? do you check that the element could be got using the interface?

Reply:
added mistakenly, removed